### PR TITLE
Place stuff in .bashrc, not .bash_profile

### PR DESCRIPTION
**Pull request by 32leaves**
fixes Java, Ruby and Rust in non-login shells (like the ones opened in GItpod).

Try here: https://gitpod.io/#https://github.com/32leaves/test-repo/tree/test-wsfull-sdkfix

### CODE REVIEW

File: full/Dockerfile [hunk 1]
```diff
@@ -253,11 +255,15 @@ RUN sudo apt-get update \
...
-RUN curl -fsSL https://sh.rustup.rs | sh -s -- -y \
+RUN cp /home/gitpod/.profile /home/gitpod/.profile_orig && \
+    curl -fsSL https://sh.rustup.rs | sh -s -- -y \
     && .cargo/bin/rustup update \
     && .cargo/bin/rustup component add rls-preview rust-analysis rust-src \
     && .cargo/bin/rustup completions bash | sudo tee /etc/bash_completion.d/rustup.bash-completion > /dev/null \
-    && .cargo/bin/rustup target add x86_64-unknown-linux-musl
+    && .cargo/bin/rustup target add x86_64-unknown-linux-musl \
+    && grep -v -F -x -f /home/gitpod/.profile_orig /home/gitpod/.profile > /home/gitpod/.bashrc.d/80-rust
+    # && mv /home/gitpod/.profile_orig /home/gitpod/.profile
```
Comment (geropl) [inline full/Dockerfile]: Leftover?

**Comment by geropl** [comment]
@32leaves Could you elaborate why the move from `profile` to `bashrc` is necessary? Just trying to understand.

**Comment by AlexTugarev** [comment]
> Could you elaborate why the move from profile to bashrc is necessary? Just trying to understand.

@geropl, in Theia we're starting `bash -i` (not `bash -l`) for new Terminals which by default will only source `.bashrc`, but not `.profile`. (`.profile` again sources `.bashrc`)

File: full/Dockerfile (hunk 1 above)
Comment (32leaves) [reply full/Dockerfile]: :+1:

**Comment by geropl** [comment]
@AlexTugarev Thank you for the explanation.

**Pull request merged**